[WIP] Create chapter on Managing personal data (#1471)
alan-turing-institute/the-turing-way: book/website/reproducible-research/rdm/personal.md

**3. Secure data transfer**
* Before considering the transfer of personal data, you should think through whether the transfer of identifiable data is necessary. 
For example, can data be de-identified or anonymised? 
* If data cannot be made unidentifiable then you must ensure you have authority to transfer the personal data, and that there are appropriate safeguards in place to protect the data before, during and after transit.
* Often your university or institute will provide solutions for secure file transfer. 
Contact you research data, privacy or IT support teams. 

**4. Encryption** 
* Encryption provides protection by ensuring that only someone with the relevant encryption key (password) will be able to access the contents.
    * Protect on disk level: Bitlocker
    * Protect on “container” level (a folder containing multiple files):  Veracrypt
    * Portable storage: Bitlocker
    * File level / Exchange information:
    * Simple method: use 7zip, and pack with a password
    * More complicated to setup: use PGP tooling (can also be used to securely send mail)

**5. Access permissions**
* management of shared folder permissions
* password protection

> **6. Anonymisation**
> Anonymisation is a process by which identifying information in a dataset is removed. 
> It is used primarily to enable data to be shared or published without revealing the confidential information it contains, while limiting the loss of information.
> * Where possible, direct identifiers (e.g. names, addresses, telephone numbers, account numbers, etc.) should be removed as soon as the identifying information is no longer needed. You can delete the data or replace it with pseudonyms. For qualitative data you should replace or generalise identifying characteristics when transcribing interviews.
> * De-identified data that can be re-identified using a linkage file (i.e. information linking data subjects to identifiable individuals) is known as pseudonymised data. NOTE: In this instance, the linkage file should be encrypted and stored securely and separately from the de-identified research data.
> o	Identification of individuals in pseudonymised or de-identified data may still be possible using combinations of indirect identifiers (e.g. age, education, employment, geographic area, medical conditions, etc). Further, data and outputs (e.g. tables of results) containing small cell counts may be potentially disclosive, particularly where samples are drawn from small populations or include cases with extreme values or relatively rare characteristics.
> o	As such, when intending to share potentially identifiable data or the outputs generated from the data, you may need to consider more advanced anonymisation techniques such as statistical disclosure control (SDC, see this handbook for more information).
> 

Reviewer: ```suggestion
**6. Anonymisation**
Anonymisation is a process by which identifying information in a dataset is removed. 
It is used primarily to enable data to be shared or published without revealing the confidential information it contains, while limiting the loss of information.
* Where possible, direct identifiers (e.g. names, addresses, telephone numbers, account numbers, etc.) should be removed as soon as the identifying information is no longer needed. 
You can delete the data or replace it with pseudonyms. 
For qualitative data you should replace or generalise identifying characteristics when transcribing interviews.
* De-identified data that can be re-identified using a linkage file (i.e. information linking data subjects to identifiable individuals) is known as pseudonymised data. 
NOTE: In this instance, the linkage file should be encrypted and stored securely and separately from the de-identified research data.
o	Identification of individuals in pseudonymised or de-identified data may still be possible using combinations of indirect identifiers (e.g. age, education, employment, geographic area, medical conditions, etc). 
Further, data and outputs (e.g. tables of results) containing small cell counts may be potentially disclosive, particularly where samples are drawn from small populations or include cases with extreme values or relatively rare characteristics.
o	As such, when intending to share potentially identifiable data or the outputs generated from the data, you may need to consider more advanced anonymisation techniques such as statistical disclosure control (SDC, see this handbook for more information).

```

* Often your university or institute will provide solutions for secure file transfer. 
Contact you research data, privacy or IT support teams for support. 

**4. Encryption** 
* Encryption provides protection by ensuring that only someone with the relevant encryption key (password) will be able to access the contents.
    * Protect on disk level: Bitlocker
    * Protect on “container” level (a folder containing multiple files):  Veracrypt
    * Portable storage: Bitlocker
    * File level / Exchange information:
    * Simple method: use 7zip, and pack with a password
    * More complicated to setup: use PGP tooling (can also be used to securely send mail)

**5. Access permissions**
* management of shared folder permissions
* password protection

**6. Anonymisation**

Anonymisation is a process by which identifying information in a dataset is removed. 
It is used primarily to enable data to be shared or published without revealing the confidential information it contains, while limiting the loss of information.

> * Where possible, direct identifiers (e.g. names, addresses, telephone numbers, account numbers, etc.) should be removed as soon as the identifying information is no longer needed. 

Reviewer 2: ```suggestion
* Where possible, direct identifiers (such as names, addresses, telephone numbers and account numbers) should be removed as soon as the identifying information is no longer needed. 
```

Author: Woops, sorry. Thank you!

You can delete the data or replace it with pseudonyms. 
For qualitative data you should replace or generalise identifying characteristics when transcribing interviews.
* De-identified data that can be re-identified using a linkage file (i.e. information linking data subjects to identifiable individuals) is known as pseudonymised data. 
NOTE: In this instance, the linkage file should be encrypted and stored securely and separately from the de-identified research data.

>   * Identification of individuals in pseudonymised or de-identified data may still be possible using combinations of indirect identifiers (e.g. age, education, employment, geographic area, medical conditions, etc). 

Reviewer 2: ```suggestion
  * Identification of individuals in pseudonymised or de-identified data may still be possible using combinations of indirect identifiers (such as age, education, employment, geographic area, and medical conditions). 
```

(rr-rdm-personal)=

## Personal data management

> **Personal data**

Author: Done! Starting out with the personal data and then mentioning the policies afterwards now

There are various policies in place in different countries to protect the rights of individuals over their personal data. 

> For example, in Europe the **GDPR** (General Data Protection Regulation) applies to the processing of personal data. 

Author: Happy to specify :D

Processing means doing anything with a person's information, including collection, storage, analysis, sharing, deletion, and destruction. 

> Please review your country's/institute's policies on managing personal data to ensure that you are up to date with the requirements of managing sensitive data. 

Author: Thanks, adjusted!

> Personal data is information about **living people** who can be identified using the data that you are processing, either directly or indirectly (for example, a person's name, address, or other unique identifier such as their Social Security number). 

Author: I have now added a direct quote from the GDPR website on this living people part: "[Data related to the deceased are not considered personal data in most cases under the GDPR](https://gdpr.eu/eu-gdpr-personal-data/)."

Indirect identifiers include health, economic, cultural or social characteristics. 
Especially when a certain combination of these identifiers can be used to identify a person, care must be taken to manage the data properly. 

> Especially sensitive data include data relating to a person's: 

Author: Thanks!

> * racial/ethnic origin

Author: Adjusted, thanks!

* political opinions
* religious/philosophical beliefs
* trade union membership
* genetic and biometric data
* physical or mental health
* sexual orientation

(rr-rdm-informed-consent)=

> **Human participants – informed consent**

Author: Good point, thanks!

Informed, voluntary and fair consent to participate in a study is very important for any research project that involves human participants. 
It is through this consent process that research participants can understand what taking part in a specific study will mean for them. 
Each person can then choose whether to participate using the consent form. 
See also the {ref}`Guide for Ethical Research<er>`

> Note that the informed consent form is considered to be personal data and should therefore be handled with the same care as other personal data. Store consent forms in a different place as the data you collect, for example a locked cabinet or an encrypted folder.

Author: Adjusted, apart from informed: that is actually a crucial word to keep in here according to the GDPR. Thanks!

> In case you cannot use a written consent form, try to make a recording of verbal consent. 

Author: It is also not always possible to make that recording, so I think it would be good to keep this formulation.

**Consent documentation should include:**

> •	a participant information sheet and

Author: Should now be adjusted, thanks!

•	a consent form signed by the participant

> The **participant information sheet** is used to inform participants about the study. The information should be  clear and easy to understand and should cover the following:

Author: Done, thanks!

* What the project is about
* What their participation will involve
* Any risks involved for participants and safeguards to minimise those risks

> * Assurances about data security and participant confidentiality: which individuals will have access to the data?

Author: Done!

> * How data will be used in the study (for published articles, reports and presentations)

Author: Adjusted!

* Proposed plans for archiving data at the end of the study and potential future secondary re-use of data
    * Tiered consent may be a solution here, by allowing the participants to choose what type of information will be shared publicly 
* Details of the organisation overseeing the research

> * Who to contact for more information about the study

Author: Thanks :P

The **consent form** is used to verify that the research participant understands and agrees to participate in the study.

> The consent form should cover the following points (but not limited to):

Author: Adjusted, thanks!

* The participant
    * has read and understood the participant information sheet

>     * has given the opportunity to ask questions

Author: Thanks!

    * understands that participation is voluntarily
    * understands that they may withdraw from the study at any time without giving reasons and without penalty
    * understands how the data will be managed, shared and archived (as detailed in the information sheet)
       * to increase the chance of your data to be reused, do not promise to delete the data but instead ask for consent to retain and share the data (see {cite}`Meyer2018personaldata`)

> * Signature and date of signing for the participant and the researcher

Author: I suppose that is not always the case but the adjusted sentence is better in any case: thanks!

Think ahead and plan how you will:

> •	collect, store and manage the data (see {ref}`Data storage and organisation<rr-rdm-storage>`)

Author: Thanks!

•	control access permissions

•	prepare data for archiving/sharing at the end of the project if possible (see {ref}`Sharing and archiving data<rr-rdm-sharing>`)

(rr-rdm-privacy)=
There are a number of strategies that you can adopt to **safeguard the privacy** of your research subjects:

**1. Data minimisation**

* If personal information isn't needed, don't collect it
* Periodically review whether you are retaining unnecessary identifying information
* When identifying information is no longer needed, safely remove, delete or destroy it

**2. Data retention limits**
* Decide how long you will retain identifiable data before removing direct identifiers, applying more complex anonymisation techniques, or deleting the data altogether

> * When deleting sensitive data you need to be aware that standard methods for deleting files (for example moving files to the recycle bin and emptying it) are not secure. These deleted files may be recovered. Use software like BleachBit to safely delete the data.

Author: I added a full stop to the end of other sentences to unify the formatting a bit here, thanks!

**3. Secure data transfer**

> * Before considering the transfer of personal data, you should think through whether the transfer of identifiable data is necessary. 

Author: Thanks!

For example, can data be de-identified or anonymised? 
* If data cannot be made unidentifiable then you must ensure you have authority to transfer the personal data, and that there are appropriate safeguards in place to protect the data before, during and after transit.
* Often your university or institute will provide solutions for secure file transfer. 

> Contact you research data, privacy or IT support teams for support. 

Author: Done! (apart from the full stop because I have been adding those)

**4. Encryption** 

> * Encryption provides protection by ensuring that only someone with the relevant encryption key (password) will be able to access the contents.

Author: Thanks!

>     * Protect on disk level: Bitlocker

Author: Added Filevault for MacOS but I'm not sure about the Linux options...

    * Protect on “container” level (a folder containing multiple files):  Veracrypt
    * Portable storage: Bitlocker

>     * File level / Exchange information:

Author: No I just screwed up the formatting, again! Thanks :D

    * Simple method: use 7zip, and pack with a password

>     * More complicated to setup: use PGP tooling (can also be used to securely send mail)

Author: Adjusted, thanks!

**5. Access permissions**
* management of shared folder permissions
* password protection

**6. Anonymisation**

Anonymisation is a process by which identifying information in a dataset is removed. 

> It is used primarily to enable data to be shared or published without revealing the confidential information it contains, while limiting the loss of information.

Author: Done, thanks!

* Where possible, direct identifiers (such as names, addresses, telephone numbers and account numbers) should be removed as soon as the identifying information is no longer needed. 
You can delete the data or replace it with pseudonyms. 
For qualitative data you should replace or generalise identifying characteristics when transcribing interviews.
* De-identified data that can be re-identified using a linkage file (for example, information linking data subjects to identifiable individuals) is known as pseudonymised data. 
NOTE: In this instance, the linkage file should be encrypted and stored securely and separately from the de-identified research data.

>   * Identification of individuals in pseudonymised or de-identified data may still be possible using combinations of indirect identifiers (such as age, education, employment, geographic area, and medical conditions). 

Reviewer 3: Need to delete the serial comma.

Author: Done, thanks!

Further, data and outputs containing small cell counts may be potentially disclosive, particularly where samples are drawn from small populations or include cases with extreme values or relatively rare characteristics.
   * As such, when intending to share potentially identifiable data or the outputs generated from the data, you may need to consider more advanced anonymisation techniques such as statistical disclosure control (SDC, see [this handbook](https://securedatagroup.org/sdc-handbook/) for more information).

> * For more information about anonymisation you can watch [this webinar by Enrico Glerean on the 'Basics of anonymization'](https://www.youtube.com/watch?v=ILXeA4fx3cI)

Reviewer 3: Maybe the link can stop at "this webinar by Enrico Glerean" to avoid the use of the -yze/-ization variant of the spelling? :P

Author: Good point, the topic is already indicated before anyway :D

**7. Sharing sensitive data**

If you plan to share or publish your data you must ensure that your data are appropriate and safe to share. 
For example, you should consider whether the data can be adequately anonymised, and whether anonymised data will remain useful (see also {ref}`Barriers to data sharing<rr-rdm-sharing>`Open Research Chapter)
After applying methods to de-identify and anonymise sensitive data, there may still be a risk of re-identification (see {cite}`Meyer2018personaldata`

> ). 

Reviewer 3: Can this be merged with the previous line?

Author: Done, thanks!

Consider applying access controls to ensure the data are shared appropriately and securely. 
This may involve finding a data repository which can provide suitable access controls (see [here](https://osf.io/tvyxz/wiki/8.%20Approved%20Protected%20Access%20Repositories/) for a list of protected Access Repositories).

**Resources**
* [Protecting sensitive data course](https://mantra.edina.ac.uk/protectionrightsandaccess) by [MANTRA](https://mantra.edina.ac.uk)
* {cite}`Meyer2018personaldata`

> * [Presentations](https://www.youtube.com/watch?v=J9kWkzK83i4&list=PLyeHH3bEQqIbgbw75gheV27nFF2ctPPpR&index=1) by [Zosia Beckles](https://youtu.be/J9kWkzK83i4), [Michele Voznick](https://youtu.be/w5v5d6r6irs) and [Tessa Darbyshire](https://youtu.be/jEFu1ykVI_I) on Responsible Data Management: Legal & Ethical Aspects as part of the [Fail to Nail it sessions](https://www.youtube.com/c/AI4ScientificDiscovery).

Reviewer 3: Full stop needs to be deleted.

* [Presentation](https://www.youtube.com/watch?v=H2mv6q4WwOU&) by Rob Gommans on GDPR and the Processing of (Identifiable) Image, Audio, and Video Data for Scientific Research Purposes
* [Presentation](https://youtu.be/_3bufely0c0) by Stephan Heunis on Brain research data and personal data privacy: practical tips to share and protect
* [Presentation](https://youtu.be/eAKhI0qde2w?t=1104) by Walter Scholger on the GDPR with resources such as informed consent templates (18:30 - 38:50)

> * [Presentation](https://www.youtube.com/watch?v=PSe2V1KTQ8w&) on handling personal data by Enrico Glerean and Päivi Lindström from Aalto University. See [here](https://www.aalto.fi/en/services/rdm-training) for the full course.

Reviewer 3: The last sentence should probably go on a new line and the full stop needs to be deleted.
